code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -38,17 +38,21 @@
 		display_text += SpecialGearRequirements()
 		to_chat(usr, display_text)
 
-/obj/item/ego_weapon/proc/CanUseEgo(mob/living/carbon/human/user)
-	if(!ishuman(user))
-		return FALSE
-	var/mob/living/carbon/human/H = user
-	for(var/atr in attribute_requirements)
-		if(attribute_requirements[atr] > get_attribute_level(H, atr))
-			to_chat(H, "<span class='notice'>You cannot use [src]!</span>")
+/obj/item/ego_weapon/proc/CanUseEgo(mob/living/user)
+	if(ishuman(user))
```

Comment:
Why would you switch the way it was done?
First we check for all things that could go wrong, not the other way around.